config/locales/diaspora/en.yml

```diff
@@ -401,25 +401,25 @@ en:
     aspects:
       title: "Aspects"
       what_is_an_aspect_q: "What is an aspect?"
       what_is_an_aspect_a: "Aspects are the way you group your contacts on diaspora*. An aspect is one of the faces you show to the world. It might be who you are at work, or who you are to your family, or who you are to your friends in a club you belong to."
       who_sees_post_q: "When I post to an aspect, who sees it?"
-      who_sees_post_a: "If you make a limited post, it will only be visible the you had placed in that aspect (or those aspects, if it is made to multiple aspects) before making the post. Contacts you have that aren’t in the aspect have no way of seeing the post, unless you’ve made it public. Limited posts will never be visible to anyone who you haven’t placed into one of your aspects."
+      who_sees_post_a: "If you make a limited post, it will only be visible to the contacts you had placed in that aspect (or those aspects, if it is made to multiple aspects) before making the post. Contacts you have that aren’t in the aspect have no way of seeing the post. Limited posts will never be visible to anyone who you haven’t placed into one of your aspects."
       contacts_know_aspect_q: "Do my contacts know which aspects I have put them in?"
       contacts_know_aspect_a: "No. They cannot see the name of the aspect under any circumstances."
       person_multiple_aspects_q: "Can I add a person to multiple aspects?"
       person_multiple_aspects_a: "Yes. Go to your contacts page and click on “My contacts”. For each contact you can use the menu on the right to add them to (or remove them from) as many aspects as you want. Or you can add them to a new aspect (or remove them from an aspect) by clicking the aspect selector button on their profile page. Or you can even just move the pointer over their name where you see it in the stream, and a “hovercard” will appear. You can change the aspects they are in right there."
       contacts_visible_q: "What does “make contacts in this aspect visible to each other” mean?"
-      contacts_visible_a: "If you check this option then contacts from that aspect will be able to see who else is in it, on your profile page under your picture. It’s best to select this option only if the contacts in that aspect all know each other, for example if the aspect is for a club or society you belong to. They still won’t be able to see what the aspect is called."
+      contacts_visible_a: "If you check this option then contacts from that aspect who are on the same pod than you will be able to see who else is in it going on your profile page and clicking the “Contacts” tab. It’s best to select this option only if the contacts in that aspect all know each other, for example if the aspect is for a club or society you belong to. They still won’t be able to see what the aspect is called."
```

Comment:
>They still won’t be able to see what the aspect is called

Definitely not an english speaker but I would have used How instead of What here.